episodes/044/README.md

```diff
@@ -1,33 +1,132 @@
+        - A quote from one of the GCP Next sessions on Knative: "Extensible at the top and pluggable at the bottom"
+    - What does serverless mean in the knative context?
+- Operator vs. developer friendliness
+    - Advertised as operator friendly
+- [Knative Logo](https://avatars2.githubusercontent.com/u/35583233?s=200&v=4)
+    - 7 sided logo with a K in it, with a 9 sided logo with an n in it.
+    - "7 of 9" inside joke. 
+
+### Installation
+
+- Used to be per-provider, PR submitted to make them generic Kubernetes instructions
+- Unfortunately EKS doesn't have the mutating admission controller flag enabled to have this work out of the box as KNative relies on [Istio automatic sidecar injection](https://istio.io/docs/setup/kubernetes/sidecar-injection/), work in progress, beta, etc.
+- Pro-tip: Pasting files into `kubectl apply` is like curling to bash, so you should check out the yaml files before blindly pasting them into a cluster. 
+- Can this be installed on on-prem install? You should be able to.
+- Pablo recommended using helm to install istio
+    - Joe doesn't normally install Tiller as it's difficult in a secure way.
+
+#### Components
+
+- Lots of large YAML files, but most of them for Knative itself is for monitoring. Look at the `-no-mon.yaml` files for smaller installs
+    - Matthew Moore: We're splitting it. "Build is required for
+      serving today, we have plans for looser coupling post 0.1"
+    -  "part of splitting off monitoring is putting the default flow on a diet"
+- We really need something like MPM to manage this
+- This is a complex beast to install and deploy. 
+  The user experience can be great, but we need to pay attention
+  to making this stuff work and maintaining it over time and
+  giving administrators insight into what is going on.
+
+##### Istio
+- Contains:
+    - Galley: Replaces Istio-auth
+    - statsd-prom-bridge: Envoy only pushes statsd metrics, so
+      need something to for Prometheus to scrape.
+- Manifest doesn't include RAM requests, but is it in its own 
+  namespace so will not have quotas set.
+- Require mutating admission controller enabled on the API server
+    - `MutatingAdmissionWebhook` added to `--enable-admission-plugins` (for 1.10+)
+- After installation, we run `kubectl label namespace default istio-injection=enabled`
+    - This will have Istio modify all pods deployed to the
+      default namespace to inject the Istio sidecars.
+
+##### Knative
+- Installs fluentd and node-exporter as a daemonset on all nodes.
+    - We need to figure out how to integrate with people's 
+      existing installations.
+- [Service (services.serving.knative.dev) is very confusing](https://github.com/knative/serving/issues/1397)
+
+### Concerns for multi-tenancy
+Want to lock down a domain names to namespaces, so that
+different tenants cannot impersonate each other. Istio
+doesn't currently allow you to lock it down.
+
+It would be great to swap Istio for other systems like
```

Comment:
I'd rephrase this as "It's be great to use the ecosystem around ingress with Knative but Istio doesn't allow that. Things like Contour would help.  Also can't do TLS with Let's Encrypt through cert-manager easily with Istio.